go.mod

```diff
@@ -1,6 +1,6 @@
 module knative.dev/eventing-kafka-broker
 
-go 1.22
+go 1.22.0
```

Comment:
why?

Reply:
This is coming from update-deps. I asked the same question in [Slack](https://cloud-native.slack.com/archives/C04LMU33V1S/p1723480072792539) and it seems we need to have this for some time: https://cloud-native.slack.com/archives/C04LMU33V1S/p1723547259859939?thread_ts=1723480072.792539&cid=C04LMU33V1S